bigquery/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 0.29.1 (unreleased)
```

Comment:
@tseaver IMO adding a section like this isn't necessary (but thanks for thinking of it).